src/pages/Tap.vue

```diff
@@ -273,10 +224,10 @@ export default {
             }
         },
         getKeyCharCode: function (event) {
-            // Try to cross platform catch the keycode
+            // Try to cross-platform catch the keycode
             // Note, there's also "event.which" (int)
             // There's also "event.key" (string), which MDN thinks is better;
-            const charCode = event.keyCode ? event.keyCode : event.charCode;
+            const charCode = event.keyCode ?? event.charCode;
```

Comment:
So we're getting null if there's no charCode?? Is that okay?

Reply:
isn't it saying "use keycode, unless null, then use charcode"?

Reply:
You're probably right. I'm still not totally au fait with the old double question mark. As you were.